pkg/http/server.go

```diff
@@ -117,7 +117,7 @@ func (s *server) useProxy(next fasthttp.RequestHandler) fasthttp.RequestHandler
 		// Add X-Forwarded-Host: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Host
 		ctx.Request.Header.Add("X-Forwarded-Host", fmt.Sprintf("%s", ctx.Host()))
 		// Add X-Forwarded-Proto: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Proto
-		ctx.Request.Header.Add("X-Forwarded-", proto)
+		ctx.Request.Header.Add("X-Forwarded-Proto", proto)
```

Comment:
I would also add unit-test for useProxy to prevent this issue later.

Reply:
Yeh I considered this but realised it was private to server so couldn't easily do it. I could pull it into a seperate package under middleware so I can test it in isolation?

Reply:
> Yeh I considered this but realised it was private to server so couldn't easily do it. I could pull it into a seperate package under middleware so I can test it in isolation?

under the same package, we can still access the private one. can we create the fake request handler like below ?
```
fakeServer = server{}
fakeRequestHandler = func(ctx *RequestCtx) {
   // assert here
}
fakeCtx = ...fake...
fakeServer.useProxy(fakeRequestHandler)(fakeCtx)
...
```

Reply:
Ah, sorry I had wrote a test but it had a typo: `Server` -> `server`. Fixed and will push.

Reply:
> Ah, sorry I had wrote a test but it had a typo: `Server` -> `server`. Fixed and will push.

I agree the naming for interface is not good here. It makes me confused too. we may need to revisit it later.